registry/proxy/scheduler/scheduler.go

```diff
@@ -155,9 +177,25 @@ func (ttles *TTLExpirationScheduler) Start() error {
 		}
 	}()
 
+	// You could paraallize this, but you'd want to make sure work was not overlapping
+	go ttles.GarbageCollect()
+
 	return nil
 }
 
+func (ttles *TTLExpirationScheduler) GarbageCollect() {
+	for {
+		select {
+		case <-ttles.garbageCollectTimer.C:
+
+			storage.MarkAndSweep(ttles.ctx, ttles.driver, ttles.registry, ttles.opts)
```

Comment:
This can be run as a separate process if desired, but is resistant to already deleted items.